synapse/models/dns.py

```diff
@@ -216,6 +216,15 @@ def getBaseModels():
                     ('udp4', {'ptype': 'inet:udp4',
                         'doc': 'The IPv4/UDP server that responded to the lookup.'}),
 
+                    ('exe', {'ptype': 'file:bytes',
```

Comment:
I was wondering if `exe` and `host` should live under `proc` here (ex: `proc:exe`), but I'd rather allow the look to be created with just host/hash info.